enhancements/single-node-production-edge-cluster-profile.md

```diff
@@ -0,0 +1,469 @@
+
+#### cluster-etcd-operator
+
+By default, `cluster-etcd-operator` will not deploy the etcd cluster
+without minimum of 3 master nodes. This can be changed by enabling
+`useUnsupportedUnsafeNonHANonProductionUnstableEtcd`.
+
+```shell
+# allow etcd-operator to start the etcd cluster without minimum of 3 master nodes
+oc patch etcd cluster --type=merge -p="$(cat <<- EOF
+
+ spec:
+   unsupportedConfigOverrides:
+     useUnsupportedUnsafeNonHANonProductionUnstableEtcd: true
+EOF
+)"
+```
+
+Even with the unsupported feature flag, `etcd-quorum-guard` still
+requires 3 nodes due to its replica count. The `etcd-quorum-guard`
+Deployment is managed by the `cluster-version-operator`, so it needs
+to be marked as unmanaged before it can be scaled down.
+
+```shell
+# tell the cluster-version-operator not to manage etcd-quorum-guard
+oc patch clusterversion/version --type='merge' -p "$(cat <<- EOF
+ spec:
+    overrides:
+      - group: apps/v1
+        kind: Deployment
+        name: etcd-quorum-guard
+        namespace: openshift-machine-config-operator
+        unmanaged: true
+EOF
+)"
+
+# scale down etcd-quorum-guard
+oc scale --replicas=1 deployment/etcd-quorum-guard -n openshift-etcd
+```
+
+#### cluster-authentication-operator
+
+By default, the `cluster-authentication-operator` will not deploy
+`OAuthServer` without minimum of 3 master nodes. This can be change by
+enabling `useUnsupportedUnsafeNonHANonProductionUnstableOAuthServer`.
+
+```shell
+# allow cluster-authentication-operator to deploy OAuthServer without minimum of 3 master nodes
+oc patch authentications.operator.openshift.io cluster --type=merge -p="$(cat <<- EOF
+
+ spec:
+   managementState: "Managed"
+   unsupportedConfigOverrides:
+     useUnsupportedUnsafeNonHANonProductionUnstableOAuthServer: true
+EOF
+)"
+```
+
+#### cluster-ingress-operator
+
+By default, the `cluster-ingress-operator` deploys the router with 2
+replicas. On a single node one will fail to start and the ingress will
+show as degraded.
+
+```shell
+# patch ingress operator to run a single router pod
+oc patch -n openshift-ingress-operator ingresscontroller/default --type=merge --patch '{"spec":{"replicas": 1}}'
+```
+
+#### machine-config-operator
+
+The machine-config-operator includes a check in code that the number
+of control plane nodes is 3. Removing this or changing the minimum is
+required to allow the operator to complete its work, for example to
+enable the realtime kernel on a single node with the appropriate
+performance profile settings.
+
+
+#### Other deployments
+
+All of these operators need to have their replica count set to 1.
+
+**Note**: The cluster-version-operator will restore these deployments
+back to 2 replicas.
+
+* openshift-authentication/oauth-openshift
+* openshift-console/console
+* openshift-console/downloads
+* openshift-monitoring/prometheus-adapter
+* openshift-monitoring/thanos-querier
+* openshift-operator-lifecycle-manager/packageserver
+
+
+
+### Risks and Mitigations
```

Comment:
given how much of a paradigm shift this is, i'd expect to see the risks articulated here...